File: BeatPlannerAndroid/src/beatplanner/ui/views/LowerMeterSpinner.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Util;
using Android.Views;
using Android.Widget;

namespace BeatPlannerAndroid
{
  public class LowerMeterSpinner : MeterSpinner
  {
    public LowerMeterSpinner(Context context) :
      base(context)
    {
    }

    public LowerMeterSpinner(Context context, int mode) :
      base(context, mode)
    {
    }

    public LowerMeterSpinner(Context context, IAttributeSet attrs) :
      base(context, attrs)
    {
    }

    public LowerMeterSpinner(Context context, IAttributeSet attrs, int defStyle) :
      base(context, attrs, defStyle)
    {

    }

    public LowerMeterSpinner(Context context, IAttributeSet attrs, int defStyle, int mode) :
      base(context, attrs, defStyle, mode)
    {

    }

    protected override int TextArrayId
    {
      get
      {
        return Resource.Array.lowerMeterValues;
      }
    }
  }
}

File: app/src/main/java/dk/aschoen/beatplanner/ui/views/LowerMeterSpinner.java

package dk.aschoen.beatplanner.ui.views;


import android.content.Context;
import android.util.AttributeSet;

import dk.aschoen.beatplanner.R;

public class LowerMeterSpinner extends MeterSpinner 
{
    public LowerMeterSpinner(Context context) {
        super(context);
    }

    public LowerMeterSpinner(Context context, int mode) {
        super(context, mode);
    }

    public LowerMeterSpinner(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public LowerMeterSpinner(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    public LowerMeterSpinner(Context context, AttributeSet attrs, int defStyle, int mode) {
        super(context, attrs, defStyle, mode);
    }

    protected int getTextArrayId() {
        return R.array.lowerMeterValues;
    }

}


